HelpSource/Classes/Dictionary.schelp

```diff
@@ -284,13 +284,23 @@ d = (a: 5, b: 7, c: 1, d: 0);
 d.asSortedArray;
 ::
 
-method::asKeyValuePairs
-Return the values in an array of alternating key value pairs.
+method::asDict
+If no arguments are passed, return itself.  This is part of the link::Reference/Key-Value-Pairs:: interface.
+argument::mergeFunc
+This argument is not used, but exists to make the method compatible.
```

Comment:
can we add "compatible with Collection.asDict" (with a link to that method)? This may be confusing out of context.